python/doc/examples/functional_modeling/viscous_fall_field_function.ipynb

```diff
@@ -0,0 +1,353 @@
GNTWFsVJIlIIljMSCrRgPL/av9vhA6e9ISX8yDHYsgxHyR1ck5K4b+ORgsdh8PC+3CjFSwFnp4D71LwqAtEJb36l0ISsMCSOdKfhC0thlcFH8Kb7eDd40epbdYnDJSq427owgZnZ9rolxHpfzpJ46mA9YAWiYmsxZzaPKAJwRz+Yr+8mWDXUGPOO1Sk90E7xAmPJNmhoJDWUARgOmHvb7BgGHQdC+fGb6QcjlTCKeZ6FM8wmvcOc4+cSO7VwU4uxLuFRbzkGUOZ/8XxDRHwsDUErIfc7mA/+FvORuG/jl1WmLEJBh6H1TlhM+C/HW66wPOJSsRwaiIh0dFoFOrtdQnNEMbbJu7cen8utc36hA6qCuwUvTiQMwcNDPMJfXslSeNlQn4PnkLFNnaQgRrcpwnB8SSLrugE67pKrHfpwuxJZ+D58yTZoaCQllAEYDrg/GFw6wnN+0DJHx9wmvuMoS4SH8Nx3139BYdJjRBjCkI1BwACec06fqQ1A8hNvjjH14fI+/2CDkLeXWDb4ZtPSeE7oXQ12HwdCrnBBD14aMFvDtzK9yFQJG0Eo36XFLeridHuJjg8sMGv+3p2itWpbdInNFKX4ZChHxcz2VFT/xNvQ+4kabwqwERgGhpesIMMVOU+jQjm+Bf7tS+vZlLV94wrNJEdPZdDVNpIo6OgkFQUAZjGuX8DxrSEcnVg9DKYJx3Ghcw0pmhMG314EJYdNOgK61BNqxBzfi0zAYkeTIpzfH2QvEfr/WVw/gusGn7L2Sh8j2iNoPt42OgNjyvBhGh4IMmBInfKQejfqW3h94ttkQKo19ei5u8l8Z37J270IpKI1DYrhirakpzQ9ee+pTlV9TPwj0jacvUkoCLQCWOs2UkGXDhW4l8AACAASURBVHlAI0I4+cV+U9uY0zp3AB0dJuI5fHGSbFBQSCsoAjANE/LWhBFNNGR3glnb4IHGn11cYxS1P6niETl2K0aPrMG9OhjJm/Ze8JjtrKAzo+Os96sPBp+6EOkD+U5ABtcUmZbCd0o2R1i8H0Zug5U6WGYKgX5yXelHXSDaP7Ut/D7RtnCBCSUYNLYFAYcu0wtX/Ek7mzVLmZbkVFQfAozUVNZN40l04m3TAJuAUKAXJuRhF+aU5z4NCOVMnP1UKvhthC1FMoTQ+F17nv+2K9E2KCikFRQBmEZ5HwJbZpQFYNE+MLOAeRwhMxnoRNmYduLIU8yWSLyd/RCjgoVjzq9iCpZY056hsY6vDwGfehDhDc5HwKzkt52PggLIedZqtYbtd8C5Mwx5BscdIXCPvD/w5QK57KBCCjOtFFL9nMxrNxDj+1F0ohSX8Uhtq2IomKEcZ8K7YTBEUUk3lTuGxIvAHMA6YBewElOc2IMZZfChHqHEvRfS1Ah2T7VHbWpMo/25ee91O9E2KCikBRQBmAbR6WByRzWB/ubM360jkwP4EcQGzjOUGhijlRu+jUB0PUJwzduYDGwW0/8+NznARnoyGVPMPxtfHwr360P4TXA+DOalUmpmCgoyGaxg/Er45TRcMIaRoeCfX05BdLsYBH95W5ZCcqNWgXs11PYWrGw6hkIhJRhATbawGEHayIHnaFuN06HtyBgegqtuBl7iSaLHagIMBEYANzDDib2YUQIf6vKeuJNkZ7GS2DvanHsZXeg8/RmGICVJtEL6RRGAaZBHt+HGeYlWYy7h9KFk72KOYYyGPnxYpxUC+p1BhIfz8rcrmKsqxfRfzngccKQpPT8bW/9eDvgIuybv+TP/ISVmpKAQO8UrwWYvaD0Opl6Fdbkh0gh8asCjDhDtl9oWfkdYGcOu2qifRrCgxwjaiaHMZyiT6UwEYaltHQAODo04GdSYXO/eUE3/I+d4kOix5gL5gHZAJBY4sR9TCuNDHcKIO+q4mLMpm5oHstOuJlMGH1aSRCukWxQBmAZxLgo77unIU/w1AO+JZDWn6UlFrDCTG217AL8/5MmK9dhk6xMTEezFWU6zl364ocXok3FFNDxsBWGe4HwQLMql6LQUFGLFyBj6TodNVyDYFvpcA+9aEPQX3MwPr5bIOSoVUoAC1vBbVVR/PGbows64sZnj/EkPKuFH4j1uyYmtY0eOva5Ekdd+1NLP5TiJiw42AbYAj5E9gWoy4MxBjHHhHrUI43qcfZvUz8Ys51u4mbRg84wvJ5VWUEirKAIwjWLxsbwvGzlPEOEMorp8wj8MBpzlfZu3hLZ5jjWtABAIljEWF4pTmzafjCcM8LgHhByFvDuVur4KaQ+nIrDuHAyZDyvOwCwL0FcF36HgXQbep53ytf9tmjvC6GIw+gJ1PaqwjnOEEEgnSqeZfYGWhYZzyDcflV48pb5hMfu+INa+REFgIbAS2AmoscSZQxiRCx9qEk7c+/xGjyxCZ/Uluj+pyoU9NxJ1fwWF1EQRgGkcAwYWc5ymFMcRO3m5oe9phAbuL5tHJgag+uDpO8sBvDjDQGZ9EiUM8HwsvN0IuTeAZa3UmImCQvyo1XIlkS3XwDIn9N0DF5qBQcgpY54OlqPXFb4xM8uAa1ZocwyXZ3nZwCWcKcoAarKVpWliX6BZmdnsvm9L/aePaCZ+ZhuXEjVOb6Ap0BN4BmiwxoUjaMnCPWoQwb1Y+0kSrJ5XnNKRd2myIzO+PgGJnYqCQqqgCMA0zlHpDnfwZyg15BPuPrD7Ce9WBqK3C8GO3oDs/VvJZEpQ+bOSby8Xwsu5kH0h2LRN6RkoKHw9OZ1hlQeMWgpb/oIJgWDoAwFr4VYBCNyhbL36pmhUsLUGGKmg1VGsoqxZyl+0YTDzGMw0uqV+vkBJhbHrGn73FrR98Ij2Yg3rOPv1wwC/AqZAZ0APaLDFmaNosOEe1YnkYax9jc207Jhsj4kugsYzA3kfpk/ChBQUUpYUE4CLFi3ixYsXKXW7/wxLVR6UIIdc9u35exj8N6JDXp41W4o1rdEi1ww+zT7ucIU+TPukQsjbzfBsOGQeA5ljzwijoJAmUamgzUDYegOy5oV+K
```

Comment:
For some reason, the Github suggestion feature does not work on this line, so here is how I would change this line: 


We see that the object first moves up and then falls down. Not all objects, however, achieve the same maximum altitude. We see that some trajectories reach a higher maximum altitude than others. Moreover, at the final time $t_{max}$, one trajectory hits the ground: $z(t_{max})=0$ for this trajectory.

Reply:
Fixed.